modules/client-common/src/main/java/org/apache/ignite/internal/client/proto/ClientMessageUnpacker.java

```diff
@@ -742,26 +741,6 @@ public UUID unpackUuid() {
         return new UUID(buf.readLong(), buf.readLong());
     }
 
-    /**
-     * Reads an {@link IgniteUuid}.
-     *
-     * @return {@link IgniteUuid} value.
-     * @throws MessageTypeException when type is not {@link IgniteUuid}.
-     * @throws MessageSizeException when size is not correct.
-     */
-    public IgniteUuid unpackIgniteUuid() {
```

Comment:
@ptupitsyn 
do we really want to drop IgniteUuid from protocol?
It may be useful e.g. for request ids, which require cluster-wide uniqueness and its generation is on hot path.

Reply:
It is trivial to bring it back if needed. For now it is dead code, let's remove.